src/drive/web/modules/drive/Toolbar/components/MoreMenu.jsx

```diff
@@ -30,45 +31,50 @@ const MoreMenu = ({
   const openMenu = useCallback(() => setMenuVisible(true))
   const closeMenu = useCallback(() => setMenuVisible(false))
 
+  const ConditionalScanner = isMobileApp() ? ScanWrapper : React.Fragment
```

Comment:
Shouldn't it be the responsibility of the ScanWrapper to render only its children on desktop ?